runtime/tests/port/j9vmemTest.c

```diff
@@ -1675,6 +1676,8 @@ j9vmem_test_reserveExecutableMemory(struct J9PortLibrary *portLibrary)
 				IDATA envp; 
 			} myDesc; 
 			myDesc.func_addr = (UDATA)memPtr;
+			myDesc.toc = 0;
```

Comment:
i am not sure if the LLVM optimizer is smart enough to treat `toc==0` specially, ending up with some weird results. The better initialization of course is to set it to the TOC from this function itself:   something like below
```
*UDATA  funcDesc = (UDATA *)(&j9vmem_test_reserveExecutableMemory);
myDesc.toc = funcDesc[1];
```